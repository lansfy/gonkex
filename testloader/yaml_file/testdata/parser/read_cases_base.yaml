- name: loader MUST generate separate test for every case
  description: original description
  method: POST
  path: /some/path
  request: requestbody
  response:
    200: someresponsebody
  cases:
    - name: subcase1
    - {} # testcase without any override
    - name: subcase3
      description: overrided description
