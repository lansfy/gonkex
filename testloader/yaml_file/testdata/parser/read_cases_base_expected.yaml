- GetName: "loader MUST generate separate test for every case #1 (subcase1)"
  GetDescription: original description
  GetMethod: POST
  Path: /some/path
  GetRequest: requestbody
  GetResponses:
    200: "someresponsebody"
  GetFileName: testdata/parser/read_cases_base.yaml
  GetLineNumber: 1
  OneOfCase: true
  FirstTestInFile: true

- GetName: "loader MUST generate separate test for every case #2"
  GetDescription: original description
  GetMethod: POST
  Path: /some/path
  GetRequest: requestbody
  GetResponses:
    200: "someresponsebody"
  GetFileName: testdata/parser/read_cases_base.yaml
  GetLineNumber: 1
  OneOfCase: true

- GetName: "loader MUST generate separate test for every case #3 (subcase3)"
  GetDescription: overrided description
  GetMethod: POST
  Path: /some/path
  GetRequest: requestbody
  GetResponses:
    200: "someresponsebody"
  GetFileName: testdata/parser/read_cases_base.yaml
  GetLineNumber: 1
  OneOfCase: true
  LastTestInFile: true
